FinniversKit/Sources/DNA/Color/ColorProvider.swift

```diff
@@ -54,168 +66,110 @@ public struct DefaultColorProvider: ColorProvider {
 
     public static let shared = DefaultColorProvider()
 
-    public var bgPrimary: UIColor {
-        .dynamicColor(defaultColor: .white, darkModeColor: .darkMilk)
-    }
+    public var bgPrimary: UIColor { .dynamicColor(defaultColor: .white, darkModeColor: .darkMilk) }
```

Comment:
I think this format makes the colors less readable, they really stand out when placed below, here you have to read to the right 🤔 and all the colors are not aligned in the same position
Not sure if it's just me though?

Reply:
No I see your point.

What about single value vars like this one for instance:

`public var accentPrimaryBlue: UIColor { .blue600 }`

would you prefer to have that expressed like this too

```
public var accentPrimaryBlue: UIColor { 
    .blue600 
}
```

For consistency's sake?

I've grown to like the "new parameter = new line" pattern so if it was me I would write it like so:

```
public var textCritical: UIColor {
    .dynamicColor(
        defaultColor: .red600,
        darkModeColor: .red400
    )
}
```

But that leads to a lot of lines, which may not be what we're after in a "set and forget" struct like this one.

TL;DR; I can live with both 😄

Reply:
I would prefer 
```
public var accentPrimaryBlue: UIColor { 
    .blue600 
}
```
so your eyes can follow the same "path", no matter if it's dynamic or not. Tbh I prefer to keep everything like it was 😄 it will also make life easier for the ones of us who currently work on colors as well (I am switching out colors in Tori, and Mohsen is making a new design system, so both of us will get a bunch of conflicts if we change the format of all colors 🙈).

Reply:
I agree for the dynamic colors. I don't have any strong opinions here so long as it's consistent within the file. The style we follow in general is function body on separate line and allow very short and simple bodies on same line, so depends on how we define short and simple. 😄

Reply:
Let's do separate lines then, so it's consistent with the style we have in general. Separate lines when there are multiple parameters is also fine by me.

Reply:
I would agree with Nina here, keep it as it is now, as small diff as possible, since we have a quite important color change being worked on for Tori. (Or wait until after that is done)

But in general we probably want to keep the changes here as few as possible because it is a temporary solution until we can go over to warp. So even making things more readable and whatever can be seen as a bit unnecessary at the time 😅

Reply:
Right...Gotcha, I will fall back to separate lines.